cylc/flow/network/schema.py

```diff
@@ -276,6 +278,16 @@ class SortArgs(InputObjectType):
     return await resolvers.get_workflows(args)
 
 
+async def subscribe_workflows(root, info, **args):
+    while True:
+        args['workflows'] = [parse_workflow_id(w_id) for w_id in args['ids']]
+        args['exworkflows'] = [parse_workflow_id(w_id) for w_id in
+                               args['exids']]
+        resolvers = info.context.get('resolvers')
+        yield await resolvers.get_workflows(args)
+        await asyncio.sleep(5.)
```

Comment:
Most examples in `graphql-ws` use a `1.` interval. However, doing so would send data to the UI too fast I think (?). At least for now, until we have the deltas and/or update the UI to handle updates selectively (i.e. update the workflow/graph data structure on the UI only if it's different.. though it'd be a bit hard for the tree at least)

Reply:
So I've set it to 5 seconds... which isn't really much different than the polling we have right now :grimacing: but one step each time :walking_man:

Reply:
Depends on the query I suppose, and whether it's only updates feed to the resolver.. i.e. 
```
query { workflows { id } }
```
would be ok.. All this will be driven by data updates in the future (which could be sub 1sec 😬 )

Reply:
If we can't just send updated content then, this is another reason why views just subscribing to exactly what they need, might be better (so not all data is sent every push)..